src/calculateRentalCost.js

```diff
@@ -4,7 +4,13 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= 7) {
+    return days * 40 - 50;
+  } else if (days >= 3) {
+    return days * 40 - 20;
+  } else {
+    return days * 40;
+  }
 }
```

Comment:
Consider using descriptive constant names for the numeric literals to clearly communicate their purpose. For example, use constants like `DAILY_RATE`, `LONG_TERM_RENTAL`, `LONG_TERM_DISCOUNT`, and `SHORT_TERM_DISCOUNT` to replace the literals `40`, `7`, `50`, and `20` respectively. This will make the code more readable and maintainable.